src/ydata_quality/drift/engine.py

```diff
@@ -106,25 +107,26 @@ def sample(self, sample: DataFrame):
         self._sample = sample
 
     @property
-    def model(self) -> Optional[ModelWrapper]:
+    def model(self) -> Optional[Union[ModelWrapper, Callable]]:
```

Comment:
model is always a `Optional[ModelWrapper]`

Reply:
solved by ca154de